Source/UseCases/Implementations/DefaultOnboardingUseCase.swift

```diff
@@ -26,6 +26,10 @@ extension DefaultOnboardingUseCase: OnboardingUseCase {
         return preferences.isTrue(.hasAcceptedTermsOfService)
     }
 
+    var hasOptedForAnalyticsPermissions: Bool {
+        return preferences.hasValue(for: .hasAcceptedAnalyticsTracking)
```

Comment:
We need to rename the preference key `hasAcceptedAnalyticsTracking` to `hasOptedForAnalyticsPermissions` or similar.

But also have a look at its use in `Tracker.swift`:

currently:
```swift
guard preferences.isTrue(.hasAcceptedAnalyticsTracking)
```

Can you can up with a naming that works for both scenarios? :)

Reply:
hasOptedForAnalyticsPermissionsAcceptingTracking :)